swagger.yaml

```diff
@@ -3131,6 +3131,32 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/release-sources:
+    get:
+      tags:
+        - versions
+      security:
+        - userAuth: [admin, read-only-admin, user]
+      operationId: v2ListReleaseSources
+      description: Retrieves openshift release sources configuration.
+      responses:
+        "200":
+          description: Success.
+          schema:
+            $ref: '#/definitions/release-sources'
+        "400":
+          description: Bad Request
+          schema:
+            $ref: '#/definitions/error'
```

Comment:
Can we really return 4xx in any case? There's no body/parameters for this endpoint so I don't see how that's an option

Reply:
What about 500? we should not hit that also